apps/aecontract/test/aecontract_SUITE.erl

```diff
@@ -1132,10 +1173,9 @@ sophia_identity(_Cfg) ->
 
 sophia_vm_interaction(Cfg) ->
     state(aect_test_utils:new_state()),
-    ForkHeight    = ?config(fork_height, Cfg),
-    RomaHeight    = ForkHeight - 1,
-    MinervaHeight = ForkHeight + 1,
-    MinervaGasPrice = aec_governance:minimum_gas_price(MinervaHeight),
+    ForkHeights   = ?config(fork_heights, Cfg),
+    RomaHeight    = maps:get(minerva, ForkHeights) - 1,
+    MinervaHeight = maps:get(fortuna, ForkHeights) - 1,
     MinerMinGasPrice= aec_tx_pool:minimum_miner_gas_price(),
     MinGasPrice   = max(MinerMinGasPrice, MinerMinGasPrice),
```

Comment:
This is weird, getting maximum of two equal numbers.

Reply:
This should probably be the unused `MinervaGasPrice` that I removed.

Reply:
I can fix this